### ISSUE TITLE
Usual Diet / Diet / population regular diet / individual regular diet

### ISSUE THREAD

**Issue by ddooley** [FoodOntology/joint-food-ontology-wg#10]
We have: 
id: ONS:0000083
label: ONS: **Usual Diet**
definition: The regular course of eating and drinking adopted by a population in a certain geographical area, or in a certain cultural setting, or following certain common eating behaviour. It is also intended as the diet a person would follow without further prescription or indications, i.e. vegetarian diet

and

id: ONS:0000080 
label: ONS: **Diet** 
definition: The regular course of eating and drinking adopted by a person.

I propose a few changes: placing both of these under a "descriptive diet" class.
renaming "diet" to "**individual regular diet**"
renaming "Usual Diet" to "**population regular diet**" , since the term "usual diet" may be a broad synonym for this or previous concept.
removing  "It is also intended as the diet a person would follow ..." from "Usual Diet" as that superimposes two concepts onto one, one of which is an average of some kind across people, and the other an average of an individual's consumption.

**Comment by FrancescoVit** [comment]
I personally agree on those changes, both at the single term definition level and placing them under the same "descriptive diet" class

> "It is also intended as the diet a person would follow ..." from "Usual Diet" as that superimposes two concepts onto one, one of which is an average of some kind across people, and the other an average of an individual's consumption.

Part of this could be used for the definition of "descriptive diet" class?

**Comment by LCCarmody** [comment]
I am good with renaming the terms. One minor nitpick, shouldn't it be 'regular individual diet' and 'regular population diet'?

This is going by order of adjectives in the English language that 'quality' goes before 'size'. I'm not sure if I am changing your meaning, but I don't think so.

**Comment by ddooley** [comment]
I thought about that but "regular" then sounds like it pertains to "individual" , not "diet".  This is the regular diet of an individual, rather than diet of a "regular individual".
Maybe that's solved with 

label: regular diet of an individual

And ditto for population?

**Comment by FrancescoVit** [comment]
Opened an issue in ONS repository for the work on those terms. 

One discussion point: in the present form, every different type of diet is a subclass of the "dietary pattern" concept, and consequently a connection between "diet" and "dietary pattern" ("dietary pattern" define "diet", or similar) can no longer hold (right?). Conceptually, I would say that this is not making real difference as mentally one could make an equivalence between a "subclass_of" property and a "define" property, but does it poses potential semantic problems? Should we consider creating a double class for each concept, one that refers to the diet and one to the pattern?

**Comment by LCCarmody** [comment]
I guess I am not sure I completely understand what you mean.  Are you saying for 'vegetarian, there should be a 'vegetarian diet pattern' and a 'vegetarian descriptive diet' ?

**Comment by FrancescoVit** [comment]
Yes, I was more or less meaning that. 
To me it is strange to have the "dietary pattern" as a super-class of the various diet concepts, as to me they are two separate facets of the same concepts, but not in a hierarchical relationship. For example, the actual definition of "dietary pattern" from the diet spreadsheet is:

_"The quantity, variety, or combination of different foods and beverage in a **diet** and the frequency with which they are habitually consumed."_

In this definition, the "diet" concept is a separated class, related to the dietary pattern by some sort of relation, which does not seem to be a hierarchical one. So, yes, if in this light I would model the vegetarian concept, I would say that there is a "vegetarian descriptive diet" which follows a "vegetarian diet pattern". Leaving the two terms separated could give the plasticity of making assertions about one or the other (i.e. the diet or the pattern) nuance of the same concept.

**Comment by ddooley** [comment]
That makes sense to me too.  A descriptive diet enumerates (lists, records) the things eaten (by an individual or group), and possibly the times they were eaten (by an individual / group).  I sense dietary pattern is expressed differently - but it would be good to have an example or two of that.  Seems like it is an abstraction of one or more descriptive diets, and has a timelessness (can be about future patterns of consumption, not just past ones, and most broadly can be prescriptive, as when one is asked to "go on a diet").

**Comment by FrancescoVit** [comment]
Depending on the context that is considered I think that "dietary pattern" could be something more tangible and defined, something measurable, while the "diet" something more abstract, resulting from the interpretation of the "dietary pattern".

To give an example (calling an opinion about this from @cyang0128 as ONE directly deals with dietary intake measurements) in the context of a dietary assessment performed by a nutritionist/researcher (and in the broader context of nutritional studies), a series of questionnaire and methods are standardized and available to measure dietary intake (i.e. 24-hours recall, Food frequency questionnaire), generally producing some "data item" as output. In cases like this, I feel that the "dietary pattern" is measured, and that its interpretation gives information about the "diet".

Also in other cases, this interpretation could be used. For example, from the dietary term spreadsheet, "DASH diet" is a type of "diet" and is defined as "Dietary Approaches to Stop Hypertension (DASH) trial, which demonstrated that consuming a diet rich in fruits, vegetables, and low-fat dairy products and reduced in saturated fat and cholesterol". A diet like this, I suppose would have some sort of "information content entity"/"data item" specifying which food should be consumed and which should be avoided, maybe in some sort of weekly table of meal suggestions. This would be the "data item" which provide information to follow the "DASH diet dietary pattern", which in turn define the "DASH diet". 

This is surely not good for every case, but I think that in a lot of cases we have a structure like this:

"data item" _specify_ "dietary pattern" _define_ "diet"

**Comment by cyang0128** [comment]
@FrancescoVit @ddooley wow, I find it is a very interesting discussion! I agree with both of you. the definition can be different in different research areas? how about having the term in FoodOn, ONS and ONE respectively, and indicating the relationships (e.g. similar to, same as, etc.)? it would be like a term in a dictionary with definitions 1, 2, 3...

**Comment by ddooley** [comment]
@cyang0128 I agree a given word or phrase can have different senses depending on research community.  I advocate though to make those differences explicit by differently named terms, and using the "x (sensu y)" pattern in the label if absolutely required, e.g. "turkey (sensu country)" vs "turkey (sensu food)" so that duplicates are minimized when searching in Ontobee or OLS .  As well, having all the diet terms in one ontology - and ONS has graciously offered to host them (i.e. FoodOn will import them from ONS) - means that a researcher can go to a single branch to see the different usages.  Its a step towards harmonization of term use across disciplines. 
However - I haven't looked at ONE yet, and of course participating in this grand interleaved ontology exercise involves coordinating with existing vocabularies and connotations.  So I'll take a close look at ONE terms.  Sorry that we didn't have you involved in the diet term discussion earlier this year!

**Comment by cyang0128** [comment]
I totally agree. it sounds nice to have all the diet terms in ONS. I would update ONE accordingly (i.e. import the ONS diet terms) as well. cheers! To facilitate your review of ONE, attached please find a screenshot, which shows a few ONE terms about the measurement and analysis of dietary intake data. @ddooley 
![image](https://user-images.githubusercontent.com/29631852/93461702-6a217c00-f8e5-11ea-8268-b94e1eb95181.png)

**Comment by FrancescoVit** [comment]
The dietary term spreadsheet for this part is practically finished, and I would be ready for its conversion to .owl with robot and then for update in ONS. 

To follow on this discussion, also considering the recent development of ONE, I would like to propose a slightly different model for the dietary classes, which can be summarized in the schema below

![immagine](https://user-images.githubusercontent.com/8997412/95063752-0f6a8b80-06ff-11eb-8a99-e9e27cbb7727.png)

On the right, the dietary assessment would be taken from ONE (now they are in green as they are also inserted in ONS); i still have to find in RO or OBI a properties to replace those "define". 

Motivation: this is more like the original implementation of ONS. To me, it was a bit strange to have "diet" class under the "dietary pattern" super-class, and not having a "diet" higher level class to be used for organization purpose. I felt like the connection between the two was in this way "opaque" using a super-sub class relationship, while a model like this, connects the data coming out form a questionnaire or other dietary assessments to a certain pattern, and hence to a certain diet.

Does it sound appropriate? Does it fit with FoodOn needs and structure?

**Comment by ddooley** [comment]
I'll take a good look at this this week.  It would help to restate the definitions of the "diet" and "dietary pattern" terms here again, to see that they don't overlap, and naturally lead to their subclasses, with a home for ONE concepts too.

**Comment by FrancescoVit** [comment]
> I'll take a good look at this this week. It would help to restate the definitions of the "diet" and "dietary pattern" terms here again, to see that they don't overlap, and naturally lead to their subclasses, with a home for ONE concepts too.

**Diet:** In nutrition, diet is the sum of food consumed by a person or other organism (with link to the wikipedia definition)
**Dietary pattern:** The quantity, variety, or combination of different foods and beverage in a diet and the frequency with which they are habitually consumed.
